ckcp/openshift_dev_setup.sh

```diff
@@ -89,6 +89,7 @@ init() {
   APP_LIST=(
             "openshift-gitops"
             "cert-manager"
+            "tekton-results"
```

Comment:
This PR covers two different install modes where the requirements are different. In a ckcp install we need to configure a postgres db instance where as on workload clusters we will be using an external database.

Reply:
Understood, so you're right this part is only relevant to local dev environment.
In such case, what about moving this task to `install_pipeline_service()`. This is where we handle deploying pipeline service in a local ckcp cluster and includes all the necessary bricks. IMO since the only purpose of this db is to be used with tekton_results then we can apply its manifets in `install_pipeline_service()`

Reply:
sure, updated now.